sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/ViewRegistryBuilder.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics;
+
+import io.opentelemetry.sdk.metrics.common.InstrumentType;
+import io.opentelemetry.sdk.metrics.view.InstrumentSelector;
+import io.opentelemetry.sdk.metrics.view.View;
+import java.util.EnumMap;
+import java.util.LinkedHashMap;
+import java.util.regex.Pattern;
+
+class ViewRegistryBuilder {
+  private final EnumMap<InstrumentType, LinkedHashMap<Pattern, View>> configuration =
+      new EnumMap<>(InstrumentType.class);
+  private static final LinkedHashMap<Pattern, View> EMPTY_CONFIG = new LinkedHashMap<>();
+
+  ViewRegistryBuilder() {
+    configuration.put(InstrumentType.COUNTER, EMPTY_CONFIG);
+    configuration.put(InstrumentType.UP_DOWN_COUNTER, EMPTY_CONFIG);
+    configuration.put(InstrumentType.VALUE_RECORDER, EMPTY_CONFIG);
+    configuration.put(InstrumentType.SUM_OBSERVER, EMPTY_CONFIG);
+    configuration.put(InstrumentType.UP_DOWN_SUM_OBSERVER, EMPTY_CONFIG);
+    configuration.put(InstrumentType.VALUE_OBSERVER, EMPTY_CONFIG);
+  }
+
+  ViewRegistry build() {
+    return new ViewRegistry(configuration);
+  }
+
+  ViewRegistryBuilder addView(InstrumentSelector selector, View view) {
```

Comment:
Completely and totally NIT: by convention builder method prefix is "with" - ie "withView".

Reply:
is it?  Our `SdkTracerProviderBuilder` uses "addSpanProcessor", and this is consistent with that.

Reply:
Typically yes, otherwise I'd not have suggested that. I believe it's usually to have naming separation between POJO (set, add) and builder (with). Additionally most builders are "fluent", and "with" sounds better when chaining methods. 
However there is no strict rule in Java world about builder naming, most things are quite "to personal taste", so as I said - completely and totally NIT :D

Reply:
we use `set` and `add` in our Builders in this project, so we should stick with that pattern.

Reply:
Definitely consistency is a good thing :)